nubis/files/nubis-ci-backup

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -l
+
+# Make sure we always run as jenkins
+if [ "$USER" != "jenkins" ]; then
+  exec su - jenkins -c "$0 $*"
+fi
+
+export PATH=/usr/local/bin:$PATH
+
+BACKUP_BUCKET=$(nubis-metadata NUBIS_CI_BUCKET)
+BACKUP_DIR=/mnt/jenkins
+
+ACTION=$1
+shift
+
+case $ACTION in
+  backup)
+    echo "Running backup"
+    if [ ! -f "$BACKUP_DIR/.initial-sync" ]; then
```

Comment:
This makes sense if the instance is replacing an existing instance. What happens when a new account is created and a CI system starts up for the first time. It looks like this logic would ensure a first run backup would never happen, or am I missing something?

Reply:
Oh, i see now. It always does a restore on boot. Got it.

Reply:
Exacrtly!